test/SpokePoolClient.v3Events.ts

```diff
@@ -536,6 +536,18 @@ describe("SpokePoolClient: Event Filtering", function () {
   });
 
   describe("SpokePoolClient: messageHash Handling", function () {
+    it("Correctly appends FundsDepositedV3 messageHash", async function () {
+      const _deposit = generateDeposit(originSpokePoolClient);
+      expect(_deposit?.args?.messageHash).to.equal(undefined);
+      await originSpokePoolClient.update(fundsDepositedEvents);
+
+      let deposit = originSpokePoolClient.getDeposit(_deposit.args.depositId);
+      expect(deposit).to.exist;
```

Comment:
Does exist equal not.undefined? I've considered using this syntax in the past but chose not to for consistency. Admittedly I didn't look into it deeply

Reply:
Docs say that `exist` checks for both `undefined` and `null`: https://www.chaijs.com/api/bdd/#method_exist

I typically only use this as a check before later re-assigning with the ! operator to force it to stop complaining about the object possibly being undefined.